frontend/class-opengraph.php

```diff
@@ -114,7 +114,25 @@ public function facebook_filter( $meta_tags ) {
 	 * @return string
 	 */
 	public function add_opengraph_namespace( $input ) {
-		return $input . ' prefix="og: http://ogp.me/ns#' . ( ( $this->options['fbadminapp'] != 0 || ( is_array( $this->options['fb_admins'] ) && $this->options['fb_admins'] !== array() ) ) ? ' fb: http://ogp.me/ns/fb#' : '' ) . '"';
+		$namespaces = array(
+			'og: http://ogp.me/ns#',
+		);
+		if ( $this->options['fbadminapp'] != 0 || ( is_array( $this->options['fb_admins'] ) && $this->options['fb_admins'] !== array() ) ) {
+			$namespaces[] = 'fb: http://ogp.me/ns/fb#';
+		}
+
+		$namespace_string = implode( ' ', array_unique( apply_filters( 'wpseo_html_namespaces', $namespaces ) ) );
```

Comment:
Good point. I wondered about that and checked both the VideoSEO source as well as WPSEO and found that hardly any of the `do_action()` or `apply_filters()` calls were documented, so kind of presumed that it wasn't a requirement.
I'll happily add documentation (late tonight - at a conference atm) if you think it will be useful/used somewhere at some point.